### ISSUE TITLE
gulp doesn't build after a failure occurs

### ISSUE THREAD

**Issue by davis** [thejameskyle/marionette-wires#39]
during development, if you introduce a syntax error into your code and save, gulp breaks and stops re-building on save

**Comment by dnewkerk** [comment]
I ran into this as well. I found a solution while studying the [gulp-starter](https://github.com/greypants/gulp-starter) example. I've switched at the moment to trying out webpack so not actively using this, but it worked recently before I made the change.

I added gulp-notify to my package.json file and added ```var notify = require('gulp-notify');``` to the top of gulpfile.js. Probably don't have to use gulp-notify, but it's pretty cool.

Then in gulpfile.js I added this callback which I borrowed from [handleErrors.js](https://github.com/greypants/gulp-starter/blob/master/gulp/util/handleErrors.js) in gulp-starter. The gulp-starter example breaks things into separate organized files, but I was just putting everything in gulpfile.js at this point.

```javascript
var handleErrors = function() {
  var args = Array.prototype.slice.call(arguments);

  notify.onError({
    title: "Compile Error",
    message: "<%= error.message %>"
  }).apply(this, args);

  // Keep gulp from hanging on this task
  this.emit('end');
};
```

Then I replaced the line ``.on('error', $.util.log)`` with ``.on('error', handleErrors)``

**Comment by VladimirPal** [comment]
What's about my PR?  This app in the main page official marionettel website, this issue is really confused users.

**Comment by idanwe** [comment]
:+1:

**Comment by thejameskyle** [comment]
Fixed in #50

**Comment by davis** [comment]
thanks!
